components/scream/src/physics/p3/p3_functions_f90.cpp

```diff
@@ -3510,20 +3521,15 @@ void p3_main_f(
     birim_d            (temp_d[counter++]),
     qv_d               (temp_d[counter++]),
     th_d               (temp_d[counter++]),
-    diag_ze_d          (temp_d[counter++]),
     diag_effc_d        (temp_d[counter++]),
     diag_effi_d        (temp_d[counter++]),
-    diag_vmi_d         (temp_d[counter++]),
-    diag_di_d          (temp_d[counter++]),
     diag_rhoi_d        (temp_d[counter++]),
     mu_c_d             (temp_d[counter++]),
-    lamc_d             (temp_d[counter++]),
+    mu_r_d             (temp_d[counter++]),
     cmeiout_d          (temp_d[counter++]),
```

Comment:
does this mean we've removed lamc and replaced with mu_r?

Reply:
Yes. I implemented the structs as described in #465 . Can you edit the description of that issue and fix the definitions of the structs to your liking?

Reply:
absolutely.  I'll do that now

Reply:
Okay, all done.  I fixed Confluence too.  Not sure how those got switched, I can't even pretend that `mu_r` and `lamc` are next to each other on the keyboard...  I would blame autocorrect, but that's probably a bridge to far.
Sorry to make sure change all these just to change them back.  Glad we caught it though.

Reply:
Thanks! And no worries--this is exactly what PRs and code reviews are for.